egs/voxpopuli/ASR/local/validate_cutset_manifest.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
```

Comment:
Could you replace this file with a symlink to its original one if no changes are done to this file?

Reply:
Actually, it is extended version of `validate_manifest.py` from Librispeech. Should I change it directly in the Librispeech recipe ? (for now i'll update it in the VoxPopuli recipe, and i'll add validation to `prepare.sh` too)